interface/patient_file/report/custom_report.php

```diff
@@ -45,6 +45,12 @@
 $PDF_OUTPUT = empty($_POST['pdf']) ? 0 : intval($_POST['pdf']);
 
 if ($PDF_OUTPUT) {
+    /* Because of the nature of this report, it is difficult to catch all the various errors possible
+     * so, turning on E_ERROR seems logical to catch memory and php errors that can't be reported due to output buffering.
+     * Try/catch will handle errors after buffer content is retrieved.
+     */
+    ini_set('display_errors', 1); // turn on diswplaying of errors.
+    error_reporting(E_ERROR); // Just fatal php errors are what concern us.
```

Comment:
@sjpadgett , This was just a thought and I'll leave this up to you. If this is needed to show errors, then feel free to leave this code in there (with or without a global; I'm ok with either).